lib/Cake/Controller/Component/SessionComponent.php

```diff
@@ -136,7 +136,13 @@ public function error() {
  * @deprecated 3.0.0 Since 2.7, use the FlashComponent instead.
  */
 	public function setFlash($message, $element = 'default', $params = array(), $key = 'flash') {
-		CakeSession::write('Message.' . $key, compact('message', 'element', 'params'));
+		$messages = (array)CakeSession::read('Message.' . $key);
+		$messages[] = [
```

Comment:
Short arrays are not compatible with PHP 5.3

Reply:
Thanks for catching that.

Reply:
It was travis, I just read the build logs.